CodeIgniter/application/views/beautyShop/allProd.twig

```diff
@@ -24,11 +24,15 @@
             {{ allBeautyProducts.name }}
             Collection: {{ allBeautyProducts.collection }}
             Price: {{ allBeautyProducts.price }}
-            Quantity: {{ allBeautyProducts.quantity }}<br/>
+            Quantity: {{ allBeautyProducts.quantity }}
+
+
+            <input type= "hidden" name= "productId" value= "{{ allBeautyProducts.id }}"/>
```

Comment:
@Katrin12323 The input and the button must be in a form, so the id from the the input could be submitted. You can leave the action attribute of the form empty for now.